tests/components/tag/test_init.py

```diff
@@ -70,7 +76,7 @@ async def test_tag_scanned(hass, hass_ws_client, storage_setup):
     assert len(result) == 2
     assert "test tag" in result
     assert "new tag" in result
-    assert result["new tag"]["last_scanned"] is not None
+    assert result["new tag"]["last_scanned"] == now.isoformat()
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)